frontend/src/components/Collections/components/CollectionsGrid/CollectionRow/index.tsx

```diff
@@ -63,30 +62,28 @@ const conditionalPopover = (values: string[]) => {
   );
 };
 
-const CollectionRow: FC<Props> = ({
-  id,
-  accessType,
-  includePrivate,
-  visibility,
-}) => {
+type DOI = {
+  doi: string;
+  link: string;
+};
+
```

Comment:
Visibility was added as a prop so the grid components could be used for the main collections view as well